lib/tasks/fix_bad_security_log_event_data.rake

```diff
@@ -0,0 +1,16 @@
+desc "Find possible duplicate acccounts"
+task fix_bad_security_log_event_data: [:environment] do
+
+  bad_to_fixed_event_data = {
+    "---\n:reason: cannot_find_user\n" =>        "{\"reason\":\"cannot_find_user\"}",
+    "---\n:reason: bad_password\n" =>            "{\"reason\":\"bad_password\"}",
+    "---\n:reason: multiple_users\n" =>          "{\"reason\":\"multiple_users\"}",
+    "---\n:reason: too_many_login_attempts\n" => "{\"reason\":\"too_many_login_attempts\"}"
+  }
+
+  bad_to_fixed_event_data.each do |bad, fix|
+    ActiveRecord::Base.connection.execute(
+      "update security_logs set event_data='#{fix}' where event_data='#{bad}'"
```

Comment:
No need to change but for future reference, you can write this as:

`SecurityLog.update_all("event_data='#{fix}' where event_data='#{bad}'")`

or even `SecurityLog.where(event_data: bad).update_all(event_data: fix)` but this option might do some conversions you don't want so it might not work (it might try to parse the bad YML in the JSON field)